client/apollo/react/src/Accordion/AccordionTagDateContainer/AccordionTagDateContainerCommon.tsx

```diff
@@ -0,0 +1,46 @@
+import { type ComponentProps, ComponentType, useMemo } from "react";
+import { Tag } from "../../Tag/TagCommon";
+
+export type AccordionTagDateContainerProps = {
+  dateLabel?: string;
+  dateProps?: ComponentProps<"time">;
+  tagLabel?: string;
+  tagProps?: ComponentProps<typeof Tag>;
+};
+
+export type AccordionTagDateContainerCommonProps =
+  AccordionTagDateContainerProps & {
+    TagComponent: ComponentType<ComponentProps<typeof Tag>>;
+  };
+
+export const AccordionTagDateContainerCommon = ({
+  dateLabel,
+  dateProps,
+  tagLabel,
+  tagProps = { variant: "warning" },
+  TagComponent,
+}: AccordionTagDateContainerCommonProps) => {
+  const isShowing = useMemo(
+    () => Boolean(tagLabel) || Boolean(dateLabel),
+    [tagLabel, dateLabel],
+  );
+
+  if (!isShowing) {
+    return null;
+  }
+
+  return (
+    <>
+      {tagLabel && (
+        <div className="af-accordion__tag-container">
+          <TagComponent {...tagProps}>{tagLabel}</TagComponent>
```

Comment:
Grâce à ce changement quand un dev voudra modifier les props de tag sans vouloir toucher au variant ce sera faisable sans supprimer le variant.
```suggestion
export const AccordionTagDateContainerCommon = ({
  dateLabel,
  dateProps,
  tagLabel,
  tagProps,
  TagComponent,
}: AccordionTagDateContainerCommonProps) => {
  const isShowing = useMemo(
    () => Boolean(tagLabel) || Boolean(dateLabel),
    [tagLabel, dateLabel],
  );

  if (!isShowing) {
    return null;
  }

  return (
    <>
      {tagLabel && (
        <div className="af-accordion__tag-container">
          <TagComponent variant="warning" {...tagProps}>{tagLabel}</TagComponent>
```